packages/tendermint-rpc/src/tendermint34/adaptors/v0-34/responses.ts

```diff
@@ -127,18 +127,24 @@ function decodeEvents(events: readonly RpcEvent[]): readonly responses.Event[] {
 
 interface RpcTxData {
   readonly code?: number;
+  readonly codespace?: string;
```

Comment:
Could you move this above code? This is a namespace schema like: `<codespace>/<code>`